modules/cron/tasks_extended.go

```diff
@@ -131,6 +131,25 @@ func registerDeleteOldActions() {
 	})
 }
 
+func registerUpdateGiteaChecker() {
+	type UpdateCheckerConfig struct {
+		BaseConfig
+		Timeout      time.Duration
+		HTTPEndpoint string
+	}
+	RegisterTaskFatal("update_checker", &UpdateCheckerConfig{
+		BaseConfig: BaseConfig{
+			Enabled:    false,
+			RunAtStart: false,
+			Schedule:   "@every 168h",
+		},
+		HTTPEndpoint: "https://dl.gitea.io/gitea/version.json",
```

Comment:
Is this the default value in case nothing is configured in app.ini? Or is the endpoint from the ini file never used? I have to admit I don't understand how the config is combined here.

Reply:
Yeah, this is the default, if there is something set for "HTTP_ENDPOINT" in config it'll override this.